docs/source/jupyter/sdk.ipynb

```diff
@@ -4,10 +4,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Python-SDK\n",
-    "\n",
-    "In order to easily and effectively communicate with a registered application in MarketPlace via Python, the Python Software Development Kit (SDK) can be used. Follow the instructions on the  [application registration](../apps/registration.md) page to register an application with MarketPlace. Once the application is registered, you will find the `client_id` in the registration output.\n",
-    "\n",
+    "# MarketPlace SDK\r\n",
+    "\r\n",
```

Comment:
It seems like this PR is introducing mixed line endings (`\r\n` vs `\n`). I thought that the `mixed-line-ending` [pre-commit hook](https://pre-commit.com/hooks.html) should fix that, but it does not seem to be the case. Can you briefly look into that?